frontend/Dockerfile

```diff
@@ -11,4 +11,4 @@ RUN npm ci
 COPY . ./
 
 EXPOSE 3000
-CMD sh start.sh
```

Comment:
What permission error? Can you explain further, I don't get what you mean.

I changed that because I wanted start.sh to run in `bash` and not with `sh`

Reply:
On #42 I left a comment about this and we also talked about when we have a virtual meeting.

Reply:
Idk in details about it, but your Docker-Envs branch used to have CMD ./start.sh, and it causes this error: 
![image](https://user-images.githubusercontent.com/60114085/140662198-d12966de-c831-43c4-a091-208083b699b1.png)

Reply:
Do you have to run this in bash instead of shell?

Reply:
does that still happen? that should not happen if start.sh has executable permissions

Reply:
we can potentially add a chmod +x before running it

Reply:
I think I tried chmod 777 kind of stuff when I was checking the docker-envs branch, which did not have good luck. I can check it again though later. Besides that, what do you think of the other two things?

Reply:
for your 3rd point it shouldnt matter because we should be overwriting it with the env var every time. I'm just gonna leave it for now because the volume is mounted currently so for everyone (except you because u are using localhost) it will rewrite package.json to have urefer-backend every time and then everyone would need to be more careful during commits.

Reply:
maybe i could make a mounted drive just for package.json, so the containers dont update it? lol

Reply:
Got it. Ive checked that the python script has been updated. I'll let you know about the permission stuff later. Great changes btw 👍

Reply:
CMD chmod +x start.sh && ./start.sh this one is working